test_util/azure_test_driver.py

```diff
@@ -58,7 +61,16 @@ def get_value(template_parameter):
     return(get_env_params()[template_parameter]['value'])
 
 
-def run():
+def get_test_config():
+    add_env = {}
```

Comment:
this doesn't seem to handle the `DCOS_AUTH_ENABLED` old case. How does that work for going through CI before and after that change?

Might be helpful to add a "Handle DCOS_AUTH_ENABLED" / oauthEnabled explicitly for the transition.